components/ButtonBar/src/MDCButtonBar.m

```diff
@@ -477,7 +477,7 @@ - (void)setButtonsTitleColor:(nullable UIColor *)color forState:(UIControlState)
 
   for (UIView *viewObj in _buttonViews) {
     if ([viewObj isKindOfClass:[UIButton class]]) {
```

Comment:
_Tip:_ I think we should match our checks and cast throughout the file, I noticed there are a couple of instances that check against `MDCButton` maybe update this to check `MDCButton` then cast to `MDCButton`.